jdi-light/src/main/java/com/epam/jdi/light/elements/complex/table/TableMatcher.java

```diff
@@ -25,23 +25,30 @@ public static TableMatcher hasValue(String value, Column column) {
                 column, "has '"+value +"'");
     }
     public static TableMatcher containsValue(String value, Column column) {
-        return new TableMatcher("/td[%s][contains(normalize-space(.),"+ Quotes.escape(value)+")]",
-                column, "contains '"+value +"'");
+        return new TableMatcher("/td[%s][contains(normalize-space(.)," + Quotes.escape(value) + ")]",
+                column, "contains '" + value + "'");
     }
 
-    private String locator;
+    private final String locator;
     private Column column;
+
     public TableMatcher setColumn(Column column) {
         this.column = column;
         return this;
     }
-    private String name;
+
+    private final String name;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)